app/helpers/application_helper.rb

```diff
@@ -14,34 +14,16 @@ def flash_messages
   end
 
   def notification_icon(subject_type)
-    case subject_type
-    when 'RepositoryInvitation'
-      'mail-read'
-    when 'Issue'
-      'issue-opened'
-    when 'PullRequest'
-      'git-pull-request'
-    when 'Commit'
-      'git-commit'
-    end
+    subject_types = {'RepositoryInvitation' => 'mail-read', 'Issue' => 'issue-opened',
```

Comment:
So instead of defining the hash here, what do you think about nesting it at the top of the module and [freezing](http://ruby-doc.org/core-2.3.3/Object.html#method-i-freeze) it?

```ruby
module ApplicationHelper
  SUBJECT_TYPES = {
    'RepositoryInvitation' => 'mail-read',
    'Issue'                => 'issue-opened',
    'PullRequest'          => 'git-pull-request',
    'Commit'               => 'git-commit'
  }.freeze

  def notification_icon(subject_type)
    SUBJECT_TYPES[subject_type]
  end
end
```